sql_generators/baseline_clients_city_seen_v1/templates/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+friendly_name: Baseline Clients City Seen
+description: |-
+  This table stores the first-seen and last-seen geo attributes for each client_id.
+  The table was initialized from stable tables (with ~2 years of retention), so the initial dates reflect the earliest/latest
+  observations within that window.  It then updates daily using live tables, updating last-seen fields for existing
+  clients and adding new clients with first-seen geo attributes that do not yet exist in the table.
+  This table should not be backfilled beyond 30 days which is the maximum retention of live tables.
+
+  Implementation Plan: https://docs.google.com/document/d/1S8yVEwJjtJy3Pd8cn_BHRhgxllVnoxKylGCpuMyqWNQ
+  Data Model Design:  https://docs.google.com/document/d/1i5SkUC5waiZGWEPu7elDIaY3IxD5jNZgJy1moAKn7g0
+owners:
+  - wichan@mozilla.com
+labels:
+  incremental: true
+  schedule: daily
+  table_type: client_level
+  dag: bqetl_clients_city_seen
+scheduling:
+  dag_name: bqetl_clients_city_seen
+  task_name: {{ app_name }}_{{ table_name }}
+  depends_on_past: true
+  date_partition_parameter: null
+  parameters:
+    - {% raw %}submission_date:DATE:{{ds}}{% endraw %}
+  depends_on:
+    - task_id: copy_deduplicate_all
+      dag_name: copy_deduplicate
+      execution_delta: 1h
```

Comment:
This isn't needed because `copy_deduplicate` is to populate the stable tables and this uses live.  Also `execution_delta` would be 3h in this case

Reply:
- One way to initialize the tables would be per sample id.  You could write a script that just runs the query per sample id
- copy_deduplicate is scheduled at 1am and bqetl_clients_city_seen is scheduled at 4am so the execution delta is 3 hours
- There's no dag for the live tables so this wouldn't depend on anything.  It would just assume data is complete after some time past midnight UTC.  This is what copy_deduplicate does

Reply:
This `depends_on` should still be removed since it's reading from the live tables